pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -1960,6 +1960,11 @@
     ::
       closed-connections
     ==
+  ::
+  ?:  ?=(%code-changed -.task)
+    ~>  %slog.[0 leaf+"eyre: code changed, throwing away old cookies"]
+    =.  authentication-state.server-state.ax  *authentication-state
+    [~ http-server-gate]
```

Comment:
To make sure that the relevant `channel-state`, timers, gall subscriptions get cleaned up, you'll want to call `+discard-channel:by-channel` for every channel in every session. (This is the part of logout logic I was thinking about when we DMed earlier.)

Reply:
Done inline in the handler for `%code-changed`. Could you take another look?